pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "gentropy"
 # !! version is managed by semantic_release
 version = "0.0.0"
-description = "Open targets Genetics Portal Python ETL"
+description = "Open Targets python framework for post-GWAS analysis"
 authors = ["Open Targets core team"]
```

Comment:
Is the genetics team part of the "core team"? If not, we should also mention them.

Reply:
they are